### PR TITLE
add x/tokenfactory

### PR DESCRIPTION
**Pull request by assafmo**

### CODE REVIEW

File: cmd/lazychaind/cmd/commands.go [hunk 1]
```diff
@@ -47,17 +45,7 @@ func initRootCmd(
...
-		rootCmd,
-		app.DefaultNodeHome,
-		newApp, appExport,
-		server.StartCmdOptions{
-			AddFlags: func(cmd *cobra.Command) {
-				rollconf.AddFlags(cmd)
-				addModuleInitFlags(cmd)
-			},
-			StartCommandHandler: rollserv.StartHandler[servertypes.Application],
-		})
```
Comment (sampocs) [inline cmd/lazychaind/cmd/commands.go]: removed the rollkit-starter-sdk in favor of the rollkit-sdk fork with modifications needed for tokenfactory

Comment (gjermundgaraba) [reply cmd/lazychaind/cmd/commands.go]: Are those changes planned to be upstreamed so we can move away from this fork eventually? I don't like very much being dependent on a fork to do other upgrades :)

Comment (assafmo) [reply cmd/lazychaind/cmd/commands.go]: The only changes are in x/bank, and they are needed to ensure x/tokenfactory works properly. I've asked osmosis if they intend to upstream it, I'll lyk when they get back to me. 

https://github.com/rollkit/cosmos-sdk/pull/119

Comment (assafmo) [reply cmd/lazychaind/cmd/commands.go]: Reverted in 4571f504f11e4ce44d179e6231a6e4435c68b7d2

**Review by gjermundgaraba — APPROVED** [review]

**Pull request merged**